tests/conftest.py

```diff
@@ -112,3 +196,18 @@ def _generate_calc_job(folder, entry_point_name, inputs=None):
         return calc_info
 
     return _generate_calc_job
+
+
+# Fixture to provide the path to the example file
+@pytest.fixture
+def example_file_path():
+    """
+    Fixture to provide the path to the example file.
+
+    Returns:
+        Path: The path to the example file.
+    """
+    return Path(__file__).resolve().parent.parent
+           / "examples"
+           / "calculations"
+           / "submit_singlepoint.py"
```

Comment:
Does this work? The new lines make me think it might not.

Reply:
nope this is old fixed already... actually wants a tuple hence the yield above....